Update dotnet-core.md (#14997)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/dotnet-core.md

### Installation

1. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
2. [Install the tracer.](#install-the-tracer)
3. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
4. [View your live data.](#view-your-live-data)

> ### Configure the Datadog Agent for APM

Reviewer: Should this part be moved to another page. I assume we should not repeat it for each tracer

Reviewer 2: Yes, we intentionally repeat it for each tracer. Some users will only use one set of instructions, we want them to get a full picture here and minimize the clicking back and forth they need to do. In this case, we duplicate content to reduce the labor of navigating docs. There are definitely grey areas to this strategy, but I think this one is solid.

[Install and configure the Datadog Agent][2] to receive traces from your instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace data on `http://localhost:8126`.

For containerized, serverless, and cloud environments:

{{< tabs >}}

{{% tab "Containers" %}}

1. Set `apm_non_local_traffic: true` in the `apm_config` section of your main [`datadog.yaml` configuration file][1].

2. See the specific setup instructions to ensure that the Agent is configured to receive traces in a containerized environment:

{{< partial name="apm/apm-containers.html" >}}
</br>

> 3. The tracing client attempts to send traces to the following:

Reviewer: I think paragraph 3 should be moved to a tracer section, because it is tracer configuration, not the agent one

> 3. After instrumenting your application, the tracing client attempts to send traces to the following:

Reviewer 2: ```suggestion
3. The tracing client attempts to send traces to the following:
```

    - The `/var/run/datadog/apm.socket` Unix domain socket by default.
    - If the socket does not exist, then traces are sent to `localhost:8126`.
    - If a different socket, host, or port is required, use the `DD_TRACE_AGENT_URL` environment variable: `DD_TRACE_AGENT_URL=http://custom-hostname:1234` or `DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket`

For more information on how to configure these settings, see [Configuration](#configuration).

{{< site-region region="us3,us5,eu,gov" >}}

4. To ensure the Agent sends data to the right Datadog location, set `DD_SITE` in the Datadog Agent to {{< region-param key="dd_site" code="true" >}}.

{{< /site-region >}}

[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
{{% /tab %}}

{{% tab "AWS Lambda" %}}

To set up Datadog APM in AWS Lambda, see [Tracing Serverless Functions][1].

[1]: /tracing/serverless_functions/
{{% /tab %}}

{{% tab "Azure App Service" %}}

To set up Datadog APM in Azure App Service, see [Tracing Azure App Service Extension][1].

[1]: /serverless/azure_app_services/
{{% /tab %}}

{{% tab "Other Environments" %}}

> Tracing is available for a number of environments including [Heroku][1], [Cloud Foundry][2], and [AWS Elastic Beanstalk][3].

Reviewer 2: ```suggestion
Tracing is available for other environments, including [Heroku][1], [Cloud Foundry][2], and [AWS Elastic Beanstalk][3].
```

> For all other environments, see the [Integrations documentation][4] for that environment and contact [Datadog support][5] if you are encountering setup issues.

Reviewer 2: ```suggestion
For all other environments, see the [Integrations documentation][4] for that environment and contact [Datadog support][5] if you encounter setup issues.
```
